windows/9.3.x/sitecore-xm/config/IdentityServer.config

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration xmlns:patch="http://www.sitecore.net/xmlconfig/">
     <sitecore>
-        <sc.variable name="identityServerAuthority">
-            <patch:attribute name="value">http://id</patch:attribute>
-        </sc.variable>
         <settings>
+            <setting name="FederatedAuthentication.IdentityServer.Authority">
```

Comment:
@joostmeijles Notice this change. Is there a consequence of changing this from `<sc.variable />` to a `<setting />` element? It works and the setting is available in `showconfig.aspx`, but is there something I'm missing by changing the element type?

Anyway, the reason for this change is to allow for `FederatedAuthentication.IdentityServer.Authority` through environment variables. It's new to me, but apparantly Sitecore replaces any occurences of `$(env:FOO)` with the value from `FOO` environment variable - if it exists.

I figured that'd be a good way forward instead of relying different patch configs. Your feedback on this is most welcome :-)